lib/thunder/bloc/thunder_state.dart

```diff
@@ -414,7 +414,7 @@ class ThunderState extends Equatable {
       openInReaderMode: openInReaderMode ?? this.openInReaderMode,
       useDisplayNames: useDisplayNames ?? this.useDisplayNames,
       markPostReadOnMediaView: markPostReadOnMediaView ?? this.markPostReadOnMediaView,
-      disableFeedFab: disableFeedFab ?? this.disableFeedFab,
+      disableFeedFab: disableFeedFab,
```

Comment:
Should the `disableFeedFab` parameter be nullable to match the others?

Reply:
I think this should be okay to leave it as is!